dev-lang/gnucobol/gnucobol-3.0_rc1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="${P/_/-}"
+
+DESCRIPTION="an open-source COBOL compiler"
+HOMEPAGE="https://open-cobol.sourceforge.io"
+SRC_URI="mirror://sourceforge/open-cobol/${MY_P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="berkdb nls readline"
+
+RDEPEND="dev-libs/gmp:0
+	sys-libs/ncurses:0
+	berkdb? ( sys-libs/db:4.8 )"
```

Comment:
Conditional runtime dependencies aren't allowed, and it looks like this is used at build-time anyway, so BDEPEND or (likely here) DEPEND.

Reply:
I took these lines from the open-cobol ebuild: https://github.com/gentoo/gentoo/blob/master/dev-lang/open-cobol/open-cobol-1.1.ebuild#L20

Reply:
Don't trust past ebuilds ;)